src/java.base/share/classes/java/lang/invoke/MethodHandleNatives.java

```diff
@@ -707,8 +706,7 @@ static boolean canBeCalledVirtual(MemberName symbolicRef, Class<?> definingClass
     static final boolean USE_SOFT_CACHE;
 
     static {
-        Properties props = GetPropertyAction.privilegedGetProperties();
```

Comment:
This is probably the last usage of Properties.

Reply:
Yes, with Sean's changes in #22497, GetPropertyAction et.al. is deleted.

Reply:
@RogerRiggs Alan may have referred to the import of `java.util.Properties`.

Reply:
Right, thanks